app/services/SpeakersService.js

```
@@ -0,0 +1,26 @@
+import DiscoveryService from './DiscoveryService.js';
+import api from './ApiService.js';
+
+const speakersUri = DiscoveryService.getUri('api-gateway', '/speakers');
+
+class SpeakersService {
+
+    getSpeaker(speakerId) {
+        return api.get(`${speakersUri}/${speakerId}`);
+    }
+
+    putSpeaker(speakerId, data) {
+        return api.put(`${speakersUri}/${speakerId}`, data);
+    }
+
+    updateRating(speakerId, newRating) {
+        return this.getSpeaker(speakerId)
+            .then(speaker => {
```

I didn't copy it, just made a typo! I got an error saying I shouldn't assign to the speaker parameter. I've adjusted so that s is being returned. Or, would you prefer I ignore the error and return speaker?